projectTemplates/ts-server/src/controllers/helloWorld.ts

```diff
@@ -0,0 +1,11 @@
+import { Router, Request, Response } from 'express';
+import { validate } from 'openapi-validator-middleware';
+import httpStatus from 'http-status-codes';
+
+const router = Router();
+
+router.get('/', validate, (req: Request, res: Response) => {
+  return res.status(httpStatus.OK).json({ hello: 'world' });
+});
+
+export { router as helloWorldController };
```

Comment:
Controllers should be seperated objects and placed under a relavant folder: 
``` js
class HelloWorldController {
	public  index (req: IRequest, res: IResponse): void {
		return res.render('pages/dashboard', {
			title: 'Home'
		});
	}
}


export default HelloWorldController;
```
now router should look 
``` js

import HelloWorldController from '../controllers/HelloWorldController'
const controller = new HelloWorldController();

router.get('/', validate, controller.index.bind(controller));

```